Integrations/integration-RTIR.yml

```diff
@@ -587,7 +587,7 @@ script:
 
     ''' EXECUTION CODE '''
 
-    tracker = rt.Rt(BASE_URL, USERNAME, PASSWORD)
+    tracker = rt.Rt(BASE_URL, USERNAME, PASSWORD, verify_cert=USE_SSL)
```

Comment:
Seems that you are doing the oposite of what you want. USE_SSL == true when unsecure is checked....

Reply:
Just checked this, It's using `USE_SSL = not demisto.params().get('unsecure', False)` To invert the user's selection. So if a user selects "Trust any certificate (Unsecure)" the parameter is `True`, but then is inverted by the `not` statement which means the verify_cert variable would equal `False`.

If the user selects `False` for "Trust any cert", it would be inverted to `True` which would mean the verify_cert would equal `True`.

It's a really strange implementation for this type of logic...